src/xtgeo/grid3d/_ecl_output_file.py

```diff
@@ -0,0 +1,77 @@
+from enum import Enum, unique
+
+
+@unique
+class TypeOfGrid(Enum):
```

Comment:
Outdated docstring I think, as it does not mention `BLOCK_CENTER`.

Reply:
Fixed!